src/stores/osimRuntime.ts

```diff
@@ -17,7 +17,8 @@ export enum OsimRuntimeStatus {
   ERROR,
 }
 
-let status = ref(OsimRuntimeStatus.INIT);
+const status = ref(OsimRuntimeStatus.INIT);
+export const versionString = ref('');
```

Comment:
Maybe I would call this variable with more unambiguous name, eg. `osimVersionString`, etc.

Reply:
resolving after reimplementing